pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/medical/records/MedicalRecordDao.java

```diff
@@ -0,0 +1,102 @@
+package org.broadinstitute.dsm.db.dao.ddp.medical.records;
+
+import org.broadinstitute.dsm.db.MedicalRecord;
+import org.broadinstitute.dsm.db.dao.Dao;
+import org.broadinstitute.dsm.util.SystemUtil;
+import org.broadinstitute.lddp.db.SimpleResult;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Arrays;
+import java.util.Optional;
+
+import static org.broadinstitute.ddp.db.TransactionWrapper.inTransaction;
+
+public class MedicalRecordDao implements Dao<MedicalRecord> {
+
+    public static final String SQL_INSERT_NEW_MEDICAL_RECORD = "INSERT INTO ddp_medical_record (institution_id, name, contact, phone, " +
+            "fax, fax_sent, fax_sent_by, fax_confirmed, fax_sent_2, fax_sent_2_by, fax_confirmed_2, fax_sent_3, fax_sent_3_by, fax_confirmed_3, " +
+            "mr_received, mr_document, mr_problem, mr_problem_text, unable_obtain, unable_obtain_text, followup_required, followup_required_text, " +
+            "duplicate, international, cr_required, pathology_present, notes, follow_ups, additional_values_json, last_changed, changed_by, mr_document_file_names, deleted) " +
+            "VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)";
+
+    private static MedicalRecordDao medicalRecordDao;
+
+    public static MedicalRecordDao of() {
+        if (medicalRecordDao == null) {
+            medicalRecordDao = new MedicalRecordDao();
+        }
+        return medicalRecordDao;
+    }
+
+    @Override
+    public int create(MedicalRecord medicalRecord) {
+        SimpleResult simpleResult = inTransaction(conn -> {
+            SimpleResult dbVals = new SimpleResult(-1);
+            try (PreparedStatement stmt = conn.prepareStatement(SQL_INSERT_NEW_MEDICAL_RECORD, Statement.RETURN_GENERATED_KEYS)) {
+                setFieldsToStatement(medicalRecord, stmt);
+                stmt.executeUpdate();
+                try (ResultSet rs = stmt.getGeneratedKeys()) {
+                    if (rs.next()) {
+                        dbVals.resultValue = rs.getInt(1);
+                    }
+                }
+            } catch (SQLException sqle) {
+                dbVals.resultException = sqle;
+            }
+            return dbVals;
+        });
+        if (simpleResult.resultException != null) {
+            throw new RuntimeException("Error while inserting medical record with institution id: " + medicalRecord.getInstitutionId(), simpleResult.resultException);
+        }
+        return (int) simpleResult.resultValue;
+    }
+
+    private void setFieldsToStatement(MedicalRecord medicalRecord, PreparedStatement stmt) throws SQLException {
+        stmt.setLong(1, medicalRecord.getInstitutionId());
```

Comment:
Optional:
I would play a little bit with reflection to make this sets automatically.